From 7d2374237c682b8d68833db1272d710e12d46157 Fri, 16 Dec 2016 13:15:14 +0100
From: Islam Amer <islam.amer@jollamobile.com>
Date: Wed, 12 Nov 2014 18:10:53 +0200
Subject: Tracker config overrides

[PATCH] [schema] change default miner initial sleep to 30 seconds to not hammer the system during bootup
Set the default delay for GraphUpdated to 350 instead of upstream default of 1000. Helps fixing JB#11570
[PATCH] [schema] change default fs miner settings to match nemo defaults
Changing the default settings in the schema should hopefully reduce issues when user resets tracker with tracker-control or similar
[defaults] match default settings with oneshot configs for throttle
[schema] change default fs miner settings to better handle android storage
[schema] exclude Android folder from indexing
Harmonize ignore folders

diff --git a/data/org.freedesktop.Tracker.Miner.Files.gschema.xml b/data/org.freedesktop.Tracker.Miner.Files.gschema.xml
index 1dfa058..92ca377 100644
--- a/data/org.freedesktop.Tracker.Miner.Files.gschema.xml
+++ b/data/org.freedesktop.Tracker.Miner.Files.gschema.xml
@@ -29,7 +29,7 @@
       <summary>Initial sleep</summary>
       <description>Initial sleep time, in seconds.</description>
       <range min="0" max="1000"/>
-      <default>15</default>
+      <default>30</default>
     </key>
 
     <key name="sched-idle" enum="org.freedesktop.TrackerMiners.TrackerSchedIdle">
@@ -46,7 +46,7 @@
       <summary>Throttle</summary>
       <description>Indexing speed, the higher the slower.</description>
       <range min="0" max="20"/>
-      <default>0</default>
+      <default>10</default>
     </key>
 
     <key name="low-disk-space-limit" type="i">
@@ -64,7 +64,7 @@
 	disables it entirely.
       </description>
       <range min="-2" max="365"/>
-      <default>-1</default>
+      <default>0</default>
     </key>
 
     <key name="removable-days-threshold" type="i">
@@ -75,7 +75,7 @@
 	maximum is 365.
       </description>
       <range min="0" max="365"/>
-      <default>3</default>
+      <default>30</default>
     </key>
 
     <key name="enable-monitors" type="b">
@@ -87,13 +87,13 @@
     <key name="enable-writeback" type="b">
       <summary>Enable writeback</summary>
       <description>Set to false to completely disable any file writeback</description>
-      <default>true</default>
+      <default>false</default>
     </key>
 
     <key name="index-removable-devices" type="b">
       <summary>Index removable devices</summary>
       <description>Set to true to enable indexing mounted directories for removable devices.</description>
-      <default>false</default>
+      <default>true</default>
     </key>
 
     <key name="index-optical-discs" type="b">
@@ -126,7 +126,7 @@
 
 	See /etc/xdg/user-dirs.defaults and $HOME/.config/user-dirs.default
       </description>
-      <default>[ '&amp;DESKTOP', '&amp;DOCUMENTS', '&amp;MUSIC', '&amp;PICTURES', '&amp;VIDEOS' ]</default>
+      <default>[ '&amp;DESKTOP', '&amp;DOCUMENTS', '&amp;DOWNLOAD', '&amp;MUSIC', '&amp;PICTURES', '&amp;VIDEOS', '$HOME/android_storage/DCIM', '$HOME/android_storage/Download', '$HOME/android_storage/Pictures', '$HOME/android_storage/Podcasts', '$HOME/android_storage/Music' ]</default>
     </key>
 
     <key name="index-single-directories" type="as">
@@ -150,13 +150,13 @@
     <key name="ignored-directories" type="as">
       <summary>Ignored directories</summary>
       <description>List of directories to avoid</description>
-      <default>[ 'po', 'CVS', 'core-dumps', 'lost+found' ]</default>
+      <default>[ 'po', 'CVS', 'core-dumps', 'lost+found', '$HOME/android_storage/Android' ]</default>
     </key>
 
     <key name="ignored-directories-with-content" type="as">
       <summary>Ignored directories with content</summary>
       <description>Avoid any directory containing a file blacklisted here</description>
-      <default>[ '.trackerignore', '.git', '.hg', '.nomedia' ]</default>
+      <default>[ '.trackerignore', 'backup.metadata', '.nomedia' ]</default>
     </key>
   </schema>
 </schemalist>


